### PR TITLE
Project

### PR DESCRIPTION
**Pull request by Munsa1**
#### Video Presentation Link (https://www.loom.com/share/20731eed735a4d34856f4232b3073bfc)
# Metrics Webapp

### Technical requirements are as follows.

-  The project is a single-page application (SPA) built with React and Redux.

-  The data retrieved from the API should be stored in the Redux store.

-  should filter the data that you retrieve from the API using a Filter stateless component.

-  Every page (the main page and the pages for each item) should have a unique route within the SPA.

-  The project should be deployed and accessible online (add a link to the online version of your app to your README file).

### Testing requirements are as follows:

-  Create unit tests for pure functions in Redux code.

-  Create integration tests for your application using the React Testing Library

Live Link (https://tender-wescoff-adf161.netlify.app/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,65 @@
...
+  - one page for the item details.
 
-### `npm test`
+## Project Description
+> Click [here](https://www.loom.com/share/6ce1cef478734b91a5feabdfaf41da22) to watch a video that gives a brief description of the project
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Demo
+> Click [here](tender-wescoff-adf161.netlify.app/) to see the live demo
```
Comment (Ghiftee) [inline README.md]: - [ ] While the live link in your PR works, I get a 404 error when I try to view the live link in your README. Kindly fix it.

File: src/index.css [hunk 2]
```diff
@@ -1,27 +1,105 @@
...
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
```
Comment (Ghiftee) [inline src/index.css]: - [ ] Kindly ensure you use the recommended font-family type that is Lato or Gill sans.
Screenshot 2022-01-07 at 13 32 18

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽  @Munsa1 

Good job working on the project so far! 
✔️ Your app works as expected.
✔️ The categories can be filtered by a given parameter.
✔️ There are no linter errors.
However, there are some issues that you need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly ensure that your video contains a highlight about a piece of the code you find interesting or are proud of making.
- [ ] As a part of the requirement is following the given design, please add the first part of the design as shown in the image below.
Screenshot 2022-01-07 at 13 39 28

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.css (hunk 2 above)
Comment (Munsa1) [reply src/index.css]: Thank you for your feedback, I have taken care of those issues

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**